### ISSUE TITLE
Tracebacks in DB editor

### ISSUE THREAD

**Issue by soininen** [spine-tools/Spine-Toolbox#1865]
Getting these while working in DB editor:

```
Traceback (most recent call last):
  File "spinetoolbox\spine_db_editor\mvcmodels\compound_parameter_models.py", line 69, in canFetchMore
    return any(self.db_mngr.can_fetch_more(db_map, self, listener=self._parent) for db_map in self.db_maps)
  File "spinetoolbox\spine_db_editor\mvcmodels\compound_parameter_models.py", line 69, in <genexpr>
    return any(self.db_mngr.can_fetch_more(db_map, self, listener=self._parent) for db_map in self.db_maps)
  File "spinetoolbox\spine_db_manager.py", line 236, in can_fetch_more
    raise RuntimeError(
RuntimeError: can't associate parent <spinetoolbox.spine_db_editor.mvcmodels.compound_parameter_models.CompoundObjectParameterDefinitionModel(0x12e1768bad0) at 0x0000012E33235880> to listener <spinetoolbox.spine_db_editor.widgets.spine_db_editor.SpineDBEditor(0x12e328fb380, name="MainWindow") at 0x0000012E359638C0> - not listening on DB map <spinedb_api.diff_db_mapping.DiffDatabaseMapping object at 0x0000012E16B35DF0>
```

```
Traceback (most recent call last):
  File "spinetoolbox\spine_db_editor\mvcmodels\metadata_table_model_base.py", line 115, in canFetchMore
    result |= self._db_mngr.can_fetch_more(db_map, fetch_parent, listener=self._db_editor)
  File "spinetoolbox\spine_db_manager.py", line 236, in can_fetch_more
    raise RuntimeError(
RuntimeError: can't associate parent <spinetoolbox.helpers.FlexibleFetchParent object at 0x0000012E33239D30> to listener <spinetoolbox.spine_db_editor.widgets.spine_db_editor.SpineDBEditor(0x12e328fb380, name="MainWindow") at 0x0000012E359638C0> - not listening on DB map <spinedb_api.diff_db_mapping.DiffDatabaseMapping object at 0x0000012E16B35DF0>
```

**Comment by manuelma** [comment]
Doing what exactly?

**Comment by soininen** [comment]
1. Clone the FlexTool project from [this GitHub repository](https://github.com/irena-flextool/flextool)
2. Open the project in Toolbox
3. Open the "Init" data store
4. Close the editor
5. Boom!

**Comment by manuelma** [comment]
I see, thank you. If they happen at closing the editor then are safe to ignore, but we need to fix them. The issue is some views keep fetching after the editor has been unregistered as listener.

**Comment by PekkaSavolainen** [comment]
Fixed by PR #1909.
